ui/apps/platform/src/routePaths.ts

```diff
@@ -104,6 +108,181 @@ export const urlEntityTypes = {
+    },
+
+    // Sidebar VulnerabilityManagement
+    [vulnManagementPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    [vulnManagementRiskAcceptancePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // VulnerabilityManagementApprovals and/or VulnerabilityManagementRequests?
+    },
+    [vulnManagementReportsPath]: {
+        readAccessPredicate: readAccessResourceNames(['VulnerabilityReports']),
+    },
+
+    // Sidebar Unexpandable2
+    [configManagementPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    [riskBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Deployment, and DeploymentExtension?
+    },
+
+    // Sidebar PlatformConfiguration
+    [clustersBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Cluster
+    },
+    /*
+    [clustersListPath]: {
+        featureFlagDependency: 'ROX_TODO', // replace conditional development rendering with backend feature flag
+        readAccessPredicate: readAccessResourceNames([]), // Cluster
+    },
+    */
+    [policiesBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Policy
+    },
+    [integrationsPath]: {
+        readAccessPredicate: readAccessResourceNames([]), // AuthPlugin is obsolete? APIToken; BackupPlugins, ImageIntegration, Notifier, SignatureIntegration; superseded by Integration?
+    },
+    [accessControlBasePathV2]: {
+        readAccessPredicate: readAccessResourceNames([]), // Access
+    },
+    [systemConfigPath]: {
+        readAccessPredicate: readAccessResourceNames(['Config']),
+    },
+    [systemHealthPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    [systemHealthPathPF]: {
+        featureFlagDependency: 'ROX_SYSTEM_HEALTH_PF',
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+
+    // Header
+    [apidocsPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    // Help Center is an external link to /docs/product
+    [userBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+};
+
+/*
+ * Evaluate feature flags and resource requirements for route paths.
+ */
+
+export type IsRenderedRoutePath = (routePath: string) => boolean;
+
+/*
+ * Higher-order function if caller needs to have predicate functions in its scope.
+ * For example, MainPath because:
+ * Body needs both isFeatureFlagEnabled and isRenderedRoutePath.
+ * NaviationSidebar needs only isRenderedRoutePath.
+ */
+export function getIsRenderedRoutePath(
+    hasReadAccess: HasReadAccess,
+    isFeatureFlagEnabled: IsFeatureFlagEnabled
+): IsRenderedRoutePath {
+    return (routePath: string) => {
+        const routeDescriptor = routeDescriptorMap[routePath];
+
+        // Delete if we replace string with RoutePath string union type.
+        if (routeDescriptor === undefined) {
+            return true;
+        }
```

Comment:
Why `true` when you can't find the route descriptor?

Reply:
Deleted this after super suggestion from Dave how to replace string with RoutePath string union type.